src/rgw/rgw_rest_s3.cc

```diff
@@ -481,6 +488,20 @@ int RGWGetObj_ObjStore_S3::send_response_data(bufferlist& bl, off_t bl_ofs,
       }
     }
 
+    if (checksum_mode) {
+      if (auto i = attrs.find("RGW_ATTR_AMZ_CHECKSUM"); i != attrs.end()) {
```

Comment:
`attrs.find(RGW_ATTR_AMZ_CKSUM)`

Reply:
you added a define for this xattr name, that's what we need to use here:
```c++
#define RGW_ATTR_AMZ_CKSUM      RGW_ATTR_PREFIX "x-amz-content-checksum"
```

Reply:
(outdated)